core/io/file_access_encrypted.h

```diff
@@ -33,7 +33,7 @@
 
 #include "core/io/file_access.h"
 
-#define ENCRYPTED_HEADER_MAGIC 0x43454447
+#define ENCRYPTED_HEADER_MAGIC 0x43'45'44'47
```

Comment:
This isn't consistent with `0xffff'ffff` used in other places.

I'm not sure either makes sense to me, if anything I'd put separators every byte (so none here, only when there's more than 8 bits).

Reply:
Two hex digits are one byte. One hex digit is 4 bits. So this is already separators every byte.

Reply:
Ah indeed! Then I guess it makes sense, though it still looks a bit overkill to me to separate every 2 or even 4 hex digits.

Reply:
I guess it's best do this on a case-by-case basis. I'll revert changes made to hexadecimal numbers, except in two situations:

- Numbers with more than 8 hexadecimal digits (will be separated every 4 digits).
- Numbers that represent binary file magic (separated every 2 digits).

Does this sound good?

Reply:
Is it useful? Splitting it into bytes doesn't make in more readable, probably better to add a `// ASCII "GDEC"` comment and uses 4 digit splits, like other places do.